### PR TITLE
add argocd image updater to dev

### PR DESCRIPTION
**Pull request by ricogu**

### CODE REVIEW

**Review by carslen — CHANGES_REQUESTED** [review]
Don't install software to productive systems (at from DevSecOps team point of view) but to DevSecOps-Testing cluster.

**Comment by ricogu** [comment]
@carslen I added this to overlay of dev cluster, so it will not affect other clusters, is my understanding right?

**Comment by carslen** [comment]
> @carslen I added this to overlay of dev cluster, so it will not affect other clusters, is my understanding right?

From our point of view the DEV environment is a productive system (of course it's not for the product teams, but for us) where we shall not install SW for internal testing purpose! That's why we setup the [devsecops-testing](https://portal.azure.com/#@catenax.onmicrosoft.com/resource/subscriptions/899135fc-19c6-47cb-82f1-0230af7b99b5/resourceGroups/cx-devsecops-testing-rg/providers/Microsoft.ContainerService/managedClusters/cx-devsecops-testing-aks/overview) cluster next to the other ones.

I won't approve PR when DEV will be target cluster for ArgoCD Image Updater evaluation.

**Comment by ricogu** [comment]
@carslen changed to testing cluster

File: environments/core/applicationsets/argo-applicationset.yaml [hunk 1]
```diff
@@ -21,7 +21,7 @@ spec:
       - cluster: devsecops-testing
         cluster-name: devsecops-testing
         overlay: overlays/devsecops-testing
-        targetRevision: onboarding-preprod
+        targetRevision: HEAD
```
Comment (ricogu) [inline environments/core/applicationsets/argo-applicationset.yaml:24]: in order not to make too many tags, I will point the testing cluster to HEAD for sometime, will make a new tag when all changes are confirm and points it to tag again

**Review by SebastianBezold — APPROVED** [review]

**Pull request merged**